visivo/models/base/parent_model.py

```diff
@@ -150,10 +156,10 @@ def name_match(trace):
     @staticmethod
     def show_dag(dag):
         import matplotlib.pyplot as pyplot
-        from networkx import planar_layout, draw_networkx
+        from networkx import spring_layout, draw_networkx
 
         options = {}
-        pos = planar_layout(dag)
+        pos = spring_layout(dag)
```

Comment:
It would be interesting to look at this for the linear project. Could be cool to add this chart to the viewer by default.

Reply:
I think we need a different layout, it is quite ugly.  :-(